src/Tools/Performance Toolkit/App/src/BCPTLine.Codeunit.al

```diff
@@ -164,7 +164,12 @@ codeunit 149005 "BCPT Line"
     var
         BCPTLogEntry: Record "BCPT Log Entry";
         BCPTRoleWrapperImpl: Codeunit "BCPT Role Wrapper"; // single instance
+        Handled: Boolean;
     begin
+        OnBeforeAddLogEntry(BCPTLine, Operation, ExecutionSuccess, Message, handled);
+
+        if Handled then
+            exit;
```

Comment:
You also need to handle the return value from this procedure; now you are returning always false.

Reply:
I don't understand what you mean. Function  AddLogEntry does not have a return value.
Or do you mean Handled (isHandled) ? That should be taken care of the subscriber to the event in BCPTTestSuite.Codeunit.

Reply:
Sorry, my bad! 

![image](https://github.com/microsoft/BCApps/assets/37966300/d825b261-5153-49e7-9b88-80b3df9e1482)

I somehow missed that the exit true/false lines are from completely different procedure 200 lines below :)

Reply:
It seems I can't mark it as resolved, please close it.